docs/docker.rst

```diff
@@ -1,74 +1,150 @@
+    in the ``jupyter-notebook`` browser window are those that exist on your
+    local machine.
+
+.. warning::
 
-    $ mkdir -p /path/to/mbuild-notebooks
-    $ cd /path/to/mbuild-notebooks
+    If you are using the container with jupyter notebooks you should use
+    the ``/home/anaconda/data`` location as the mount point inside the container;
+    this is the default notebook directory.
 
-2. Define an entry-point script. Inside `mbuild-notebooks` in your local file system create a file called :code:`dir_entrypoint.sh` and paste the following content.
+Running Python scripts in the container
+---------------------------------------
+Jupyter notebooks are a great way to explore new software and prototype
+code. However, when it comes time for production science, it is often
+better to work with python scripts. In order to execute a python script
+(``example.py``) that exists in the current working directory of your
+local machine, run:
 
 .. code-block:: bash
 
-    #!/bin/sh
+    $ docker run --mount type=bind,source=$(pwd),target=/home/anaconda/data mosdef/mbuild:latest "python data/test.py"
 
-    chown -R anaconda:anaconda /home/anaconda/mbuild-notebooks
+Note that once again we are ``bind`` mounting the current working directory
+to ``/home/anaconda/data``. The command we pass to the container is
+``python data/test.py``. Note the prefix ``data/`` to the script; this is because
+we enter the container in the home folder (``/home/anaconda``), but our script
+is located under ``/home/anaconda/data``.
 
-    su anaconda -s /bin/sh -l -c "jupyter-notebook --no-browser --ip="0.0.0.0" --notebook-dir /home/anaconda/mbuild-notebooks"
+.. warning::
 
-3. Run docker image for `mbuild`
+    Do not bind mount to ``target=/home/anaconda``. This will cause errors.
+
+
+If you don't require a Jupyter notebook, but just want a Python interpreter,
+you can run:
+
+.. code-block:: bash
+
+    $ docker run --mount type=bind,source=$(pwd),target=/home/anaconda/data mosdef/mbuild:latest python
+
+If you don't need access to any local data, you can of course drop the
+``--mount`` command:
 
 .. code-block:: bash
 
-    $ docker run -it --name mbuild -p 8888:8888 --entrypoint /home/anaconda/mbuild-notebooks/dir_entrypoint.sh -v /home/umesh/mbuild-notebooks:/home/anaconda/mbuild-notebooks mosdef/mbuild:latest
+    $ docker run mosdef/mbuild:latest python
+
 
+Different mBuild versions
+-------------------------
+Instead of using ``latest``, you can use the image ``mosdef/mbuild:stable``
+for most recent stable release of mBuild.
 
 Cleaning Up
 -----------
-You can remove the created container by using the following command:
+You can remove the *container* by using the following command. 
 
 .. code-block:: bash
 
     $ docker container rm mbuild
 
+The *image* will still exist on your machine. See the tutorials at the
+top of this page for more information.
+
+.. warning::
+
+    You will not be able to start a second container with the same name
```

Comment:
We can also add a section on stopping a container and restarting it, also information on detached mode might help as well :).

Reply:
Yes, I was trying to balance between too little/too much general docker information.

I agree about detached mode, but _should_ we talk about restarting? My (very limited) understanding of containers is that the prototypical workflow is _not_ to save any data into the containers; persistent data should be bind or volume mounted into the container. I'm open to hearing the opposite opinion. In this manner the container really just operates as an "application". Thoughts?

Reply:
Oh I meant that in the context of starting a second container with the same name.